### PR TITLE
Deleted manifest data, added contributing doc

### PR DESCRIPTION
**Pull request by jaepass**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ # Contributing
```
Comment (robbieaverill) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing
```

**Comment by ScopeyNZ** [comment]
Huh? We need the manifest! That powers the cli?

**Comment by jaepass** [comment]
heyhey Guy sorted it out sorry this was a big my bad, a while back when i was working on the `chec/resources` repo, i thought i had cloned a copy of the original `chec/example-stores` but instead renamed and built on top of it. so now just reverting that mistake and added the manifest back into a new `chec/example-stores` repo https://github.com/chec/example-stores

**Comment by ScopeyNZ** [comment]
Yep all good. @robbieaverill explained it to me 🙂

**Pull request merged**